README.md

```diff
@@ -309,7 +309,7 @@ To run local build of UTA you can follow these steps...
            --name uta-build \
            --volume .:/opt/repos/uta \
            --volume /System/Volumes/Data/locus/data/test-uta-22:/temp \
-           --network=host uta-build:latest
+           --network=host uta:latest
```

Comment:
What do we want to name the various containers we are working with. So far there are three.

1. SeqRepo, used to create the Docker volume: seqrepo
2. UTA build, container where the update occurs: uta
3. UTA Postgresql database, where previous and new versions exist: uta-db

I have been calling `uta`, `uta-build` to distinguish it from the database container.

Reply:
I ended up calling it `uta-update` in `update.sh` for the same reason. I had updated the README before that.